sass/style-short.scss

```diff
@@ -1,5 +1,6 @@
 #logo {
-  height: 100px;
+  background-image: url(../css/images/gravity.coast.cassette.web.short.png);
```

Comment:
Yeah, I thought that we re-arranged to account for this yesterday, but I was finding that the shortened logo image wasn't loading.  The logo images were in the css/images folder because the logo was being displayed by being the background image of a div... so from the css file or sass file, it would have to go UP one directory to see the images folder.   Child's play for you, but how do we work around this?